### PR TITLE
Proof TVCRESTARTS

### LINKED ISSUE #270: Proof TVCRESTARTS
Proof read transcribed code of [TVCRESTARTS](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/TVCRESTARTS.agc) against scans

Lines: 272
Page: 0956—0960

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### PR DESCRIPTION
**Pull request by Gaboso**
This should close #270

Some content fixes were mostly indentation/spacing and period/comma fixes.

### CODE REVIEW

**Comment by wopian** [comment]
Currently very busy, so wont be able to take a look at this until start of next week 😢

File: Comanche055/TVCRESTARTS.agc [hunk 1]
```diff
@@ -34,47 +34,47 @@
 
 # Page 956
 # NAME....TVCRESTART PACKAGE, CONSISTING OF REDOTVC, ENABL1, 2, CMDSOUT, PHSCHK2, ETC.
-# LOG SECTION....TVCRESTARTS			SUBROUTINE....DAPCSM
```
Comment (oldmud0) [inline Comanche055/TVCRESTARTS.agc]: :heavy_check_mark:

File: Comanche055/TVCRESTARTS.agc [hunk 2]
```diff
@@ -34,47 +34,47 @@
...
+# LOG SECTION...TVCRESTARTS			SUBROUTINE...DAPCSM
 # MODIFIED BY SCHLUNDT				21 OCTOBER 1968
 #
 # FUNCTIONAL DESCRIPTION....
 #
 #      *RESTART-PROCESS THE TVC DAPS, INCLUDING PITCHDAP, YAWDAP,
```
Comment (oldmud0) [inline Comanche055/TVCRESTARTS.agc]: Should be `*RESTART-PROOFS`. Technically, these bullet points should go into the indentation, not beside it (see [scan](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0956.jpg)).

File: Comanche055/TVCRESTARTS.agc [hunk 3]
```diff
@@ -34,47 +34,47 @@
...
-#	TVCEXECUTIVE, ROLLDAP, TVCINIT4, TVCDAPON, AND CSM/.M V46 SWTCHOVR.
```
Comment (oldmud0) [inline Comanche055/TVCRESTARTS.agc]: :heavy_check_mark:

File: Comanche055/TVCRESTARTS.agc [hunk 4]
```diff
@@ -34,47 +34,47 @@
...
+#	TVCEXECUTIVE, ROLLDAP, TVCINIT4, TVCDAPON, AND CSM/LM V46 SWTCHOVR.
 #
```
Comment (oldmud0) [inline Comanche055/TVCRESTARTS.agc]: GAP printout doesn't have newlines between bullet points, so these must go.

File: Comanche055/TVCRESTARTS.agc [hunk 5]
```diff
@@ -131,8 +130,8 @@
...
 # OUTPUT....
 #
-#      *PITCH AND YAW TVC DAP COPYCYCLES COMPLETED IF INTERRUPTED
-#      *TVCEXECUTIVE COMPLETED IF INTERRUPTED
+#      *PITCH AND YAW TVC DAP COPYCYCLES COMPLETED IF INTERUPTED
+#      *TVCEXECUTIVE COMPLETED IF INTERUPTED
```
Comment (oldmud0) [inline Comanche055/TVCRESTARTS.agc]: We're doing our job guys :heavy_check_mark:

File: Comanche055/TVCRESTARTS.agc [hunk 6]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
 		TCF	CHKSTRK
 		TCF	CHKSTRK
 		CCS	A		# A CONTAINS THE DIMINISHED ABSOLUTE OF
-		TC	+3		# TVCPHASE (-2 BECOMES +1. -1 BECOMES +0)
+		TC	+3		# TVCPHASE (-2 BECOMES +1, -1 BECOMES +0)
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 7]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
 
-		TC	POSTJUMP	#	REPEAT TVC INITIALIZATION
-		CADR	MRCLEAN		#	(DO NOT RETURN)
+		TC	POSTJUMP	#      REPEAT TVC INITIALIZATION
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 8]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
+		CADR	MRCLEAN		#      (DO NOT RETURN)
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 9]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
 
-	+3	TC	IBNKCALL	#	REPEAT CSM/LM V46 SWITCH-OVER
-		CADR	SWICHOVR +5	#	(RETURN TO CHECK FOR STROKE TEST)
+	+3	TC	IBNKCALL	#      REPEAT CSM/LM V46 SWITCH-OVER
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 10]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
+		CADR	SWICHOVR +5	#      (RETURN TO CHECK FOR STROKE TEST)
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 11]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
 
 CHKSTRK		CCS	STROKER		# CHECK FOR STROKE TEST IN PROGRESS
 		TCF	TSTINITJ	# YES, KILL IT
 		TCF	+2		# NO, PROCEED
 		TCF	TSTINITJ	# YES, KILL IT
 
-	+4	TC	POSTJUMP	#	IF POSITIVE OR ZERO, RESTART AT
-		CADR	TVCINIT4	#		TVCINIT4 (ZEROS TVCPHASE, AND
-					#		CALLS TVC DAPS VIA DAPINIT)
+	+4	TC	POSTJUMP	#      IF POSITIVE OR ZERO, RESTART AT
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 12]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
+		CADR	TVCINIT4	#           TVCINIT4 (ZEROS TVCPHASE, AND
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 13]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
+					#           CALLS TVC DAPS VIA DAPINIT)
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 14]
```diff
@@ -216,30 +215,30 @@ PHSCHK2		CCS	TVCPHASE	# CHECK TVCPHASE AGAIN
...
 FINCOPY		INDEX	TVCPHASE	# PICK UP THE APPROPRIATE COPYCYCLE
 		CAF	TVCCADR
 		TCR	ISWCALL		# RE-ENTER THE COPYCYCLE, RETURN AT END
 		TCF	ENABL1		# NOW PREPARE THE OUTCOUNTERS
 
 TRIM/CMD	EXTEND			# TVCDAPON INITIALIZATION NOT COMPLETED,
 # Page 960
-		DCA	PACTOFF		#	EG. P,YCMD MAY NOT BE SET.  SET...
+		DCA	PACTOFF		#      EG. P,YCMD MAY NOT BE SET.  SET...
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️ Alignment
✨ Double space after `EG.`

File: Comanche055/TVCRESTARTS.agc [hunk 15]
```diff
@@ -249,23 +248,23 @@ TSTINITJ	CAF	ZERO		# DISABLE STROKE TEST (-0 SHOWS PRIOR V68)
 		TCF	CHKSTRK +4
 
 EXRSTRT		INDEX	TVCEXPHS	# TVCEXECUTIVE RESTARTS....GO TO
-		CAF	TVCEXADR	#	APPROPRIATE RESTART POINT
+		CAF	TVCEXADR	#      APPROPRIATE RESTART POINT
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 16]
```diff
@@ -249,23 +248,23 @@ TSTINITJ	CAF	ZERO		# DISABLE STROKE TEST (-0 SHOWS PRIOR V68)
...
 		INDEX	A
 		TCF	0
 
 # TVC RESTART TABLES.... ORDER IS REQUIRED.  HI-ORDER WORDS ONLY, OF 2CADRS, SINCE BBCON IS ALREADY THERE.
 
 TVCADDR		=	TVCCADR		# TABLE OF CADRS, UNUSED LOCS FOR GENADRS
 TVCCADR		GENADR	ENABL2		# (FOR T5 CALL, UNUSED TABLE LOC)
-	+1	CADR	PCOPY 	+1	# PITCH COPYCYCLE
+	+1	CADR	PCOPY +1	# PITCH COPYCYCLE
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 17]
```diff
@@ -249,23 +248,23 @@ TSTINITJ	CAF	ZERO		# DISABLE STROKE TEST (-0 SHOWS PRIOR V68)
...
 	+2	GENADR	CMDSOUT		# (FOR T5 CALL, UNUSED TABLE LOC)
-	+3	CADR	YCOPY 	+1	# YAW COPYCYCLE
+	+3	CADR	YCOPY +1	# YAW COPYCYCLE
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

File: Comanche055/TVCRESTARTS.agc [hunk 18]
```diff
@@ -249,23 +248,23 @@ TSTINITJ	CAF	ZERO		# DISABLE STROKE TEST (-0 SHOWS PRIOR V68)
...
 TVCEXADR	OCT	37772		# (UNUSED TABLE LOC, FILL WITH 60MS, T5)
 	+1	GENADR	EXECCOPY +1	# TVCEXECUTIVE RESTART POINTS (ORDERED)
 	+2	GENADR	1SHOTCHK
 	+3	GENADR	TEMPSET
 	+4	GENADR	CORSETUP
-	+5	GENADR	CORCOPY	+1
+	+5	GENADR	CORCOPY +1
```
Comment (wopian) [inline Comanche055/TVCRESTARTS.agc]: ✔️

**Comment by chrislgarry** [comment]
@oldmud0 when youre done making the few adjustments we can merge this in. Thanks guys for all your hard word @oldmud0 @wopian @Gaboso

**Comment by oldmud0** [comment]
/me tries really hard not to sit on this PR

**Comment by oldmud0** [comment]
What were those comments you deleted?

**Comment by wopian** [comment]
>What were those comments you deleted?

All the ✔️-only review comments

**Comment by chrislgarry** [comment]
@Gaboso I will pick this up if you have no time to implement the requested changes. Please let me know. I want to give everyone an opportunity to contribute.

**Pull request closed without merging**